lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -3570,4 +3570,9 @@ public function getStringLiteralQuoteCharacter()
     {
         return "'";
     }
+
+    public function getSequenceMetadataSQL(string $sequenceName, string $schemaName) : string
```

Comment:
It doesn't belong here. Please implement `getListSequencesSQL()` for the new platform so that it returns sequence names and metadata same as it's done in the `OraclePlatform`. The only platform which needs to be able to provide metadata for one individual sequence is PostgreSQL < 10 because it doesn't have sequence metadata available via a view.